core/drivers/crypto/versal/include/ipi.h

```diff
@@ -0,0 +1,96 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) Foundries Ltd. 2022 - All Rights Reserved
```

Comment:
"All rights reserved" is useless (https://en.wikipedia.org/wiki/All_rights_reserved#Obsolescence), so its use in new files is not recommended.

Reply:
ok